stdlib/3.4/enum.pyi

```diff
@@ -7,11 +7,14 @@ class Enum:
     def __dir__(self) -> List[str]: ...
     def __format__(self, format_spec: str) -> str: ...
     def __hash__(self) -> Any: ...
+    def __iter__(self) -> Iterator['Enum']: ...
```

Comment:
I agree that `__iter__` should be a classmethod, and have modified my PR as such.

However, I do not agree that the return type should be `Iterator[Any]`. While it is not as narrow as it could be, `Iterator[Enum]` is accurate for Enums and subclasses, and is substantially narrower that `Iterator[Any]`